components/brave_wallet/api/asset_ratio.idl

```diff
@@ -64,50 +64,9 @@ namespace asset_ratio {
   //     "message": "OK",
   //     "result": []
   //
-  dictionary TokenInfoPayload {
-    // A list with result records.
-    TokenInfoResult[] result;
-  };
-
-
-  // A token info dictionary. Looks like this:
-  // {
-  //   "payload": {
-  //     "status": "1",
-  //     "message": "OK",
-  //     "result": [
-  //       {
-  //         "contractAddress": "0xdac17f958d2ee523a2206206994597c13d831ec7",
-  //         "tokenName": "Tether USD",
-  //         "symbol": "USDT",
-  //         "divisor": "6",
-  //         "tokenType": "ERC20",
-  //         "totalSupply": "39828710009874796",
-  //         "blueCheckmark": "true",
-  //         "description": "Tether gives you the joint benefits of open...",
-  //         "website": "https://tether.to/",
-  //         "email": "support@tether.to",
-  //         "blog": "https://tether.to/category/announcements/",
-  //         "reddit": "",
-  //         "slack": "",
-  //         "facebook": "",
-  //         "twitter": "https://twitter.com/Tether_to",
-  //         "bitcointalk": "",
-  //         "github": "",
-  //         "telegram": "",
-  //         "wechat": "",
-  //         "linkedin": "",
-  //         "discord": "",
-  //         "whitepaper": "https://path/to/TetherWhitePaper.pdf",
-  //         "tokenPriceUSD": "1.000000000000000000"
-  //       }
-  //     ]
-  //   },
-  //   "lastUpdated": "2021-12-09T22:02:23.187Z"
-  // }
```

Comment:
The old proxy implementation in brave-intl/bat-ratios added this wrapper.  That's not included in the new HAProxy implementation, so I've removed this from the parsing logic.